lib/view/user_settings/user_switch_dialog.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import 'package:hakondate/router/routes.dart';
+import 'package:hakondate/view/component/dialog/hakondate_dialog/hakondate_dialog.dart';
+import 'package:hakondate/view_model/multi_page/user/user_view_model.dart';
+import 'package:hakondate/view_model/single_page/daily/daily_view_model.dart';
+
+class UserSwitchDialog extends ConsumerWidget {
+  const UserSwitchDialog({required this.id, super.key});
+
+  final int id;
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return HakondateDialog(
+      title: const Text('選択中のお子様の変更'),
+      body: const Text('選択中のお子様を変更しますか？'),
```

Comment:
変更だと編集と意味合いが似ているので以下の表現の方が良いな。
```suggestion
      body: const Text('選択中のお子様を切り替えますか？'),
```